### DISCUSSION TITLE
expect result on streamer

### DISCUSSION THREAD

**Discussion by foxnight2** [paullouisageneau/libdatachannel#1046] [Q&A] [answered]
https://github.com/paullouisageneau/libdatachannel/tree/master/examples/streamer

What is the expect result for running streamer.exe ?

The image is my result of running http://127.0.0.1:8080/.

![image](https://github.com/paullouisageneau/libdatachannel/assets/142980812/225ac032-ddf0-4185-839e-fd9041592b9c)

**Comment by chobie** [comment, marked as answer]
The build directory of cmake does not contain any HTML files in the 'example' folder. 
when you want to start a Python server with the 'streamer' exmaple. you have to launch it in the `libdatachannel/example/streamer` directory.

```
chobie streamer % pwd
/Users/chobie/github/libdatachannel/examples/streamer
chobie streamer % ls *html
index.html  # http server will show this file.

chobie streamer % pwd
/Users/chobie/github/libdatachannel/build/examples/streamer
chobie streamer % ls
cmake_install.cmake  CMakeFiles  Makefile  streamer 
# the build directory. index.html does not exist. so python http server will show directory index like you uploaded image.
```

**Comment by chobie** [comment]
Essentially, `example` program are provided in a manner similar to the `copy-paste` example. 
When you paste the SDP message outputted from the console into the form in the browser and execute it, an answer SDP is returned. 
then,  you paste that answer into the console, you can see example result.

It's written straightforwardly, so please take a look at the code for more details.
